app/editor/src/components/form/wysiwyg/styled/Wysiwyg.tsx

```diff
@@ -27,6 +28,7 @@ export const Wysiwyg = styled.div<IWysiwygProps>`
 
   .ql-editor {
     min-height: 25rem;
+    height: ${(props) => props.hasHeight && '42rem'};
```

Comment:
What happens if `hasHeight == false`?

Reply:
if hasHeight == false, then we don't assign anything here, and will keep using whatever the inherited value is. Thanks

Reply:
Check the outputted css that Styled components create.  I suspect it will include an invalid css element because it will be `height: ;`

Reply:
Currently based on the testing it seems to be working fine as expected, thanks.